npc/pre-re/warps/cities/alberta.txt

```diff
@@ -0,0 +1,10 @@
+//===== rAthena Script ======================================= 
+//= Alberta City Portals
+//===== Changelogs: ==========================================
+//= 1.0 First Version. [JohnnyPlayy]
+//============================================================
+
+//= Portal - ( Rooms )
+//============================================================
+alberta,134,38,0	warp	Room_Entrance#alberta-01	1,1,alberta_in,70,141
+alberta_in,73,141,0	warp	Room_Exit#alberta-01	1,1,alberta,137,37
```

Comment:
The change of name was intended ?
```suggestion
alberta,134,38,0	warp	alb02	1,1,alberta_in,70,141
alberta_in,73,141,0	warp	alb02-1	1,1,alberta,137,37
```

Reply:
I wrote before looking at how the portals were. That's why it's like this.